src/test/java/com/knighttodo/knighttodo/utilis/TestUtils.java

```diff
@@ -0,0 +1,22 @@
+package com.knighttodo.knighttodo.utilis;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+
+import java.io.IOException;
+
+public class TestUtils {
+    private TestUtils() {
+    }
+
+    public static byte[] convertObjectToJsonBytes(final Object object) throws IOException {
```

Comment:
We can use ObjectMapper here, but I'll tell you about it later, when we'll do micro-service architecture

Reply:
Okay